<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminSetting</class>
 <widget class="QDialog" name="AdminSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="settingLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kenyan Coffee</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>61</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="oldLabel">
      <property name="font">
       <font>
        <family>Kenyan Coffee</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Old Password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="oldInput"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="userLabel">
      <property name="font">
       <font>
        <family>Kenyan Coffee</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>New Username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="userInput"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="passwordLabel">
      <property name="font">
       <font>
        <family>Kenyan Coffee</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>New Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="passwordInput">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QCheckBox" name="showBox">
      <property name="text">
       <string>Show Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
